src/A1_csv_to_json.py

```diff
@@ -359,10 +360,112 @@ def create_json_from_csv(input_directory, filename, parameters, storage=False):
             "No %s" % filename + " assets are added because all "
             "columns of the csv file are empty."
         )
-
-    for column in df:
+    df_copy = df
+    for column in df_copy:
         if column != "unit":
             column_dict = {}
+            # the storage columns are checked for the right parameters,
```

Comment:
Could you format the following lines with f"{}" instead of "%s" (this part is new so it should not create conflicts :))